src/services/decoders/JsonlDecoder/index.ts

```diff
@@ -122,6 +122,29 @@ class JsonlDecoder implements Decoder {
         return results;
     }
 
+    getLogEventIdxByTimestamp (timestamp: number): number {
+        let low = 0;
+        let high = this.#logEvents.length - 1;
+        let result = -1;
```

Comment:
let's see if the suggestion at https://github.com/y-scope/yscope-log-viewer/pull/152/files#r1890816021 makes sense. If not (i.e., we want to keep the return type as `number` rather than `Nullable<number>`), let's create a const for `-1` so we can avoid magic numbers.